astrobee/config/localization.config

```diff
@@ -29,11 +29,11 @@ brisk_num_ransac_iterations = 100
 brisk_early_break_landmarks = 100
 brisk_histogram_equalization = 1
 brisk_check_essential_matrix = false
-brisk_essential_ransac_iterations = 2000
+brisk_essential_ransac_iterations = 0
```

Comment:
is this really right? (just triple checking...)

Reply:
@rsoussan ?

Reply:
Oh yeah that's fine, is ignored either way since it's set to false in the line above